bin/ci

```diff
@@ -84,7 +84,10 @@ prepare_build() {
 
   on_linux docker pull "jhass/crystal-build-$ARCH"
 
-  on_osx brew install llvm crystal-lang
+  on_osx brew install llvm@5 crystal-lang
```

Comment:
`crystal-lang` itself currently depends on `llvm@5`, so it's not necessary to `install` it.

We do need to link it, though - we can get the currently depended `llvm` version via `jq` with `brew info --json=v1 crystal-lang | jq '.[].dependencies | .[] | select(startswith("llvm"))'`. Not sure if overkill, but avoiding re-specifying the LLVM version we need seems sweet.

Reply:
nice one, is there `jq` installed already?

Reply:
It worked perfectly ;)